### ISSUE TITLE
bug(a11y): Screen reader reads list items in reversed order on android

### ISSUE THREAD

**Issue by OtavioStasiak** [RocketChat/Rocket.Chat.ReactNative#6295]
### Describe the Bug

When navigating the RoomView using TalkBack, the screen reader reads the list of messages in reverse order starting from the last message and going backwards instead of following the visual top-to-bottom order.

### Steps to Reproduce

- Open the app.
- Go to the room screen.
- Turn on TalkBack.
- Navigate through the screen.

### Expected Behavior

The screen reader should announce list items in the same visual order they appear on the screen.

### Actual Behavior

On Android devices, the screen reader reads from bottom to top.

### Rocket.Chat Server Version

7.5

### Rocket.Chat App Version

7.59.0

### Device Name

Xiaomi Mi A3

### OS Version

Android 11 +
